dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/StatsTaskListGenerator.java

```diff
@@ -73,11 +73,10 @@ static StatsQuery create(String name, StatsSource tool) {
     }
 
     @Nonnull
-    Task<?> toTask() throws IOException {
+    String queryText() throws IOException {
```

Comment:
Nested `StatsQuery` class is now used in the external `StatsJdbcTask` class. I'd suggest exporting `StatsQuery` as a separate top level class for better readability.

Reply:
Done